src/containers/costume-tab.jsx

```diff
@@ -173,7 +177,7 @@ class CostumeTab extends React.Component {
             bitmapResolution: item.info.length > 2 ? item.info[2] : 1,
             skinId: null
         };
-        this.handleNewCostume(vmCostume);
+        this.handleNewCostume(vmCostume, true /* fromCostumeLibrary */);
```

Comment:
I think this line probably needs to be added to whatever handles surprise costumes as well.

Reply:
Oh sorry, the diff made it look like the line was being added to `handleNewCostume`